modules/material/themes/material/default/other_mfas.twig

```diff
@@ -0,0 +1,27 @@
+{% if otherOptions|length > 0 %}
+  <div layout-children="column" child-spacing="center">
+    {# used type=button to avoid form submission on click since this is just used to display the ul #}
```

Comment:
So nothing happens on the click event? Would it be better to not use a button? I realize this is just copied from the old file.

Reply:
I don't really know. It doesn't make sense to me either.